File: freecodecamp/bonfire/8-confirm-the-ending.rb
def end?(string, target)
  string[-target.length, target.length] == target
end

p end?('Bastian', 'n')
p end?('Connor', 'n')
p end?('Walking on water and developing software from a specification are easy if both are frozen', 'specification')
p end?('He has to give me a new name', 'name')

File: freecodecamp/bonfire/8-confirm-the-ending.js
function end(str, target) {
  return str.search(new RegExp(target + '$', 'g')) >= 0 ? true : false;
}

console.log(end("Bastian", "n"));
console.log(end("Connor", "n"));
console.log(end("Walking on water and developing software from a specification are easy if both are frozen", "specification"));
console.log(end("He has to give me a new name", "name"));
